docs/data/routes/docs/nextjs/page-routing/switching-to-ssr/en.md

```diff
@@ -3,4 +3,15 @@ name: switching-to-ssr
 routeTemplate: ./data/component-templates/article.yml
 title: Switching to SSR
 ---
-# Walkthrough: Switching to SSR
\ No newline at end of file
+# Walkthrough: Switching to SSR
+
+By default, the Next.js sample application uses a route optimized for SSG. However, we have also provided a working SSR route you can use: `\pages_examples\[[..path]].SSR.tsx`.
```

Comment:
I like to use github links for any files I'm referencing from the sample app. My thinking is that people might be referencing this material when they're further into implementation, after they've deleted sample app material.

Reply:
Good idea, added.